tests/test_fs.py

```diff
@@ -62,3 +71,26 @@ def test_initialization(monkeypatch: MonkeyPatch, temporary_lakectl_config: str)
     assert config.host == "http://lakefs.hello/api/v1"
     assert config.username == "my-user"
     assert config.password == "my-password"
+
+
+@pytest.mark.skipif(lakefs_version < Version("0.14"), reason="requires lakefs>=0.14.0")
+def test_request_config(
```

Comment:
While this is a fine test case, I would also like to exercise the code path that passes the request args to the FS constructor to validate the logic that transforms the passed args.